src/test/behave_tests/features/validation_util.py

```diff
@@ -1,5 +1,6 @@
+from typing import List
 
-def str_to_list(string):
+def str_to_list(string: str) -> List[List[str]]:
```

Comment:
do we want to run mypy against this?

Reply:
I think we should. I've already integrated mypy and flake8 into tox for all the files in the features directory.